data_acquisition/clear_expired_jobs.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import os
+import sys
+import psycopg2
+import getpass
+import logging
+
+url_index = 2
+id_index = 0
+
+# Currently just makes a GET request to the URLs in the DB and returns true if the status code is 200
```

Comment:
returns true if the status code is NOT 200 meaning that the job has been expired.

Reply:
Oop my bad on that typo in my comment. I'll fix that